README.md

```diff
@@ -7,8 +7,9 @@ replication capabilities with utilities to set up standby servers, monitor
 replication, and perform administrative tasks such as failover or switchover
 operations.
 
-The most recent `repmgr` version (5.4.1) supports all PostgreSQL versions from
-10 to 16.
+The most recent `repmgr` version (5.5.x) supports all PostgreSQL versions from
+13 to 17. Despite it could be used with some older one, some features might not
+be available, however, it's strongly recommended to use the latest version possible.
```

Comment:
```suggestion
The most recent `repmgr` version (5.5.x) supports all PostgreSQL versions from
13 to 17. Despite it could be used with some older ones, some features might not
be available, however, it's strongly recommended to use the latest version.
```